master/buildbot/www/hooks/github.py

```diff
@@ -266,8 +278,14 @@ def _get_pr_files(self, repo, number):
             verify=self.verify,
         )
         res = yield http.get(url)
-        data = yield res.json()
-        return [f["filename"] for f in data]
+        if 200 <= res.code < 300:
+            data = yield res.json()
+            return [f["filename"] for f in data]
+        else:
```

Comment:
I think this else is not needed, we can just fall through o the end of the function.

Reply:
Also, coverage results show that this error path is not covered by any tests. Can we add a test that covers it?

Reply:
Same